package.json

```diff
@@ -79,7 +79,7 @@
     "@leafygreen-ui/menu": "18.0.0",
     "@leafygreen-ui/modal": "6.1.2",
     "@leafygreen-ui/palette": "3.4.4",
-    "@leafygreen-ui/popover": "7.2.2",
+    "@leafygreen-ui/popover": "11.0.0",
     "@leafygreen-ui/radio-box-group": "12.0.0",
     "@leafygreen-ui/radio-group": "10.0.0",
     "@leafygreen-ui/segmented-control": "5.0.0",
```

Comment:
(writing this mostly as a reminder to myself) The new tooltips make hovering on the waterfall icons pretty funky, it's possible we need to reduce the delay

Reply:
Can we ticket this? Cailin also mentioned that highlighting tooltips on the waterfall is too slow for her liking. So if it's even slower we should address it before releasing it.

Reply:
done EVG-18374